addons/mail/static/src/messaging/component/followers_menu/followers_menu.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.FollowersMenu" owl="1">
+        <div class="o_FollowersMenu">
+            <t t-if="thread.isCurrentPartnerFollowing">
+                <button class="o_FollowersMenu_unfollowButton btn btn-link" type="button" t-att-disabled="props.isDisabled" t-on-click="_onClickUnfollow" t-on-mouseenter="_onMouseEnterUnfollow" t-on-mouseleave="_onMouseLeaveUnfollow" t-att-class="{ 'o-following': !state.hasUnfollowHover, 'o-unfollow': state.hasUnfollowHover }">
```

Comment:
Can't hover stuff just be css? We need to store that?

Reply:
t-att-class after class, "type button" on button seems useless

Reply:
Don't know if you talked about that with @alexkuhn but it was just css thing before he changed so I think it's best to ask him why we need to store that

Reply:
Ok I will wait the answer, which should then be commented in the code to avoid further doubts about it.

Reply:
CSS `:hover` is good enough when special style on hover only affects CSS.
Here, hover makes changes to template, therefore behaviour on hover should be handled by the component itself.